### ISSUE TITLE
Are you going to update the Bagnon addon soon??

### ISSUE THREAD

**Issue by Tammii** [tullamods/Bagnon#604]

**Comment by Torsin** [comment]
Did you try downloading the following zipped up fix for this from the Project Issue #585:

[Bagnon_73_Ready.zip](https://github.com/tullamods/Bagnon/files/1264452/Bagnon_73_Ready.zip)

**Comment by Tammii** [comment]
Ok, I downloaded the ZIP file, but it still doesn't work.  What folder do I
have to download it to?  I have saved it to my WoW folder on my C: drive,
but I'm not a computer programmer.  :-)  Should I move it to my WoW
Interface folder maybe?  Or the WoW Data folder?  I don't have a folder
called Addons in my WoW folder on my C: drive.

Thanks for you quick response though, I'm so frustrated without this
amazing Bagnon addon!  :-))

**Comment by Tammii** [comment]
It works now!  Saved it to my Addon folder in my Interface folder!  :-)
 Thanks so much!  I'm a happy customer now and will spread the news!

Have a great weekend!

Tammy

**Comment by aashe4** [comment]
I deleted the existing Bagnon folders (5) from my Interface\Addons folder, downloaded the zip, turned off Twitch so it didn't try to "help", extracted the 5 folders to Interface\Addons, and started the game. I was still experiencing the same issues. 

I then did the same thing except I pulled over one file at a time. Same issue. I cannot open my bags at all and when I attempt to I get this:

Date: 2017-09-02 15:23:05
ID: 2
Error occured in: Global
Count: 1
Message: ...s\Bagnon\external\Wildpants\components\frameBase.lua line 22:
   Usage: PlaySound(soundKitID, optional["SFX","Music","Ambience", "Dialog" or "Master"], [forceNoDuplicates, runFinishCallback])
Debug:
   [C]: PlaySound()
   ...s\Bagnon\external\Wildpants\components\frameBase.lua:22: OnHide()
   ...s\Bagnon\external\Wildpants\components\inventory.lua:17:
      ...s\Bagnon\external\Wildpants\components\inventory.lua:16
   [C]: Hide()
   ..\FrameXML\UIParent.lua:2944: HideUIPanel()
   ...ace\AddOns\Bagnon\external\Wildpants\core\frames.lua:63:
      ...ace\AddOns\Bagnon\external\Wildpants\core\frames.lua:59
   (tail call): ?
   (tail call): ?
   ...s\Bagnon\external\Wildpants\features\autoDisplay.lua:117: ToggleBackpack()
   ..\FrameXML\MainMenuBarBagButtons.lua:54: BackpackButton_OnClick()
   [string "*:OnClick"]:4:
      [string "*:OnClick"]:1
Locals:
(*temporary) = "igBackPackClose"

AddOns:
  Swatter, v7.4.5714 (TasmanianThylacine)
  WowheadLooter, v70204
  Altoholic, vr182
  Atlas, vv1.43.01
  AtlasArena, vv1.07.01
  AtlasBattlegrounds, vv1.43.00
  AtlasBurningCrusade, vv1.43.00
  AtlasCataclysm, vv1.43.00
  AtlasClassicWoW, vv1.43.00
  AtlasClassOrderHalls, vv1.43.00
  AtlasDungeonLocs, vv1.43.00
  AtlasLegion, vv1.43.00
  AtlasMistsofPandaria, vv1.43.00
  AtlasOutdoorRaids, vv1.43.00
  AtlasScenarios, vv1.43.00
  AtlasTransportation, vv1.43.00
  AtlasWarlordsofDraenor, vv1.43.00
  AtlasWorldEvents, vv3.23
  AtlasWrathoftheLichKing, vv1.43.00
  AtlasLoot, vv8.06.00
  AtlasLootBurningCrusade, vv8.06.00
  AtlasLootCataclysm, vv8.06.00
  AtlasLootClassic, vv8.06.00
  AtlasLootLegion, vv8.06.00
  AtlasLootMistsofPandaria, vv8.06.00
  AtlasLootWarlordsofDraenor, vv8.06.00
  AtlasLootWrathoftheLichKing, vv8.06.00
  AtlasMajorCitiesEnhanced, vv1.15
  AucAdvanced, v7.4.5714 (TasmanianThylacine)
  AucFilterBasic, v7.4.5714 (TasmanianThylacine)
  AucFilterOutlier, v7.4.5714.5459(7.4/embedded)
  AucMatchUndercut, v7.4.5714.5585(7.4/embedded)
  AucStatHistogram, v7.4.5714 (TasmanianThylacine)
  AucStatiLevel, v7.4.5714 (TasmanianThylacine)
  AucStatPurchased, v7.4.5714 (TasmanianThylacine)
  AucStatSales, v7.4.5714.5598(7.4/embedded)
  AucStatSimple, v7.4.5714 (TasmanianThylacine)
  AucStatStdDev, v7.4.5714 (TasmanianThylacine)
  AucStatWOWEcon, v7.4.5714.5613(7.4/embedded)
  AucUtilAHWindowControl, v7.4.5714.5575(7.4/embedded)
  AucUtilAppraiser, v7.4.5714.5704(7.4/embedded)
  AucUtilAskPrice, v7.4.5714.5599(7.4/embedded)
  AucUtilAutoMagic, v7.4.5714.5686(7.4/embedded)
  AucUtilCompactUI, v7.4.5714.5694(7.4/embedded)
  AucUtilEasyBuyout, v7.4.5714.5576(7.4/embedded)
  AucUtilFixAH, v7.4.5714 (TasmanianThylacine)
  AucUtilItemSuggest, v7.4.5714.5651(7.4/embedded)
  AucUtilPriceLevel, v7.4.5714.5633(7.4/embedded)
  AucUtilScanButton, v7.4.5714.5631(7.4/embedded)
  AucUtilScanFinish, v7.4.5714.5588(7.4/embedded)
  AucUtilScanProgress, v7.4.5714.4979(7.4/embedded)
  AucUtilScanStart, v7.4.5714.5347(7.4/embedded)
  AucUtilSearchUI, v7.4.5714.5710(7.4/embedded)
  AucUtilSimpleAuction, v7.4.5714.5705(7.4/embedded)
  AucUtilVendMarkup, v7.4.5714.4828(7.4/embedded)
  Babylonian, v5.1.DEV.332(/embedded)
  BagBrother, v
  Bagnon, v7.2.7
  Bartender4, v4.7.10
  BeanCounter, v7.4.5714 (TasmanianThylacine)
  Configator, v5.1.DEV.406(/embedded)
  Coordinates, v1.23
  DataStore, vr63
  DataStoreAchievements, vr45
  DataStoreAgenda, vr25
  DataStoreAuctions, vr48
  DataStoreCharacters, vr40
  DataStoreContainers, vr56
  DataStoreCrafts, vr77
  DataStoreCurrencies, vr35
  DataStoreGarrisons, vr21
  DataStoreInventory, vr50
  DataStoreMails, vr49
  DataStorePets, vr42
  DataStoreQuests, vr42
  DataStoreReputations, vr40
  DataStoreSpells, vr30
  DataStoreStats, vr29
  DataStoreTalents, vr59
  DBMCore, v7.3.1-4-gcf6ddd6
  DBMDefaultSkin, v
  DBMStatusBarTimers, v
  DebugLib, v5.1.DEV.337(/embedded)
  Enchantrix, v7.4.5714 (TasmanianThylacine)
  EnchantrixBarker, v7.4.5714 (TasmanianThylacine)
  HandyNotes, vv1.4.15
  HandyNotesDraenorTreasures, v1.20a
  HandyNotesLegionRaresTreasures, v1.01b
  Informant, v7.4.5714 (TasmanianThylacine)
  LibExtraTip, v5.12.DEV.423(/embedded)
  Pawn, v2.2.13
  Recount, vv7.2.5f
  SlideBar, v7.4.5714 (TasmanianThylacine)
  Stubby, v7.4.5714 (TasmanianThylacine)
  SuramarPortals, v1.01
  TipHelper, v5.12.DEV.405(/embedded)
  Titan, v5.12.3.70300
  TitanArtifactPower, v1.9.0
  TitanBag, v5.12.3.70300
  TitanClock, v5.12.3.70300
  TitanGold, v5.12.3.70300
  TitanLocation, v5.12.3.70300
  TitanLootType, v5.12.3.70300
  TitanPerformance, v5.12.3.70300
  TitanRepair, v5.12.3.70300
  TitanVolume, v5.12.3.70300
  TitanXP, v5.12.3.70300
  TomTom, vv70200-1.0.0
  WeakAuras, v2.4.21
  WorldQuestTracker, v
  WoWDBProfiler, v
  BlizRuntimeLib_enUS v7.3.0.70300 <none>
  (ck=f24)
